akvo/rsr/models/result/indicator.py

```diff
@@ -201,6 +201,9 @@ def is_parent_indicator(self):
         """
         return self.child_indicators.count() > 0
 
+    def is_cumulative(self):
+        return self.cumulative and self.measure != PERCENTAGE_MEASURE
```

Comment:
Probably should add a comment to explain this

Reply:
Fixed